doc/newsboat.asciidoc

```diff
@@ -1130,7 +1130,7 @@ Dialog List:<<dialogs-title-format,+dialogs-title-format+>>:%N %V - Dialogs
 [frame="all", grid="all", format="dsv", options="header", cols="30,70"]
 |======================================================================
 Identifier:Meaning
-[[common-title-format-N]]<<common-title-format-N,+N+>>:Name of the program, i.e. "newsboat"
+[[common-title-format-N]]<<common-title-format-N,+N+>>:Name of the program, i.e. "Newsboat"
```

Comment:
We'll need to change some code to make this true, so for now, please revert this change.